### ISSUE TITLE
Parsing error: Unexpected token, expected , (Fatal)

### ISSUE THREAD

**Issue by gfelot** [eslint/eslint#9208]
**Tell us about your environment**

* **ESLint Version:** v4.6.0
* **Node Version:** v8.4.0
* **npm Version:** v5.4.0

**What parser (default, Babel-ESLint, etc.) are you using?**
Babel-ESLint
**Please show your full configuration:**
```json
{
    "env": {
      "browser": true,
      "node": true
    },
    "parser": "babel-eslint",
    "plugins": [
        "react"
    ],
    "extends": [
        "plugin:react/recommended",
        "eslint-config-airbnb"
    ],
    "rules": {
      "indent": [1, 4],
      "react/jsx-indent": [1, 4],
      "react/jsx-indent-props": [1, 4],
      "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
      "jsx-quotes": ["error", "prefer-single"],
      "comma-dangle": ["error", {
          "arrays": "never",
          "objects": "never",
          "imports": "never",
          "exports": "never",
          "functions": "ignore"
        }]
    },
    "parserOptions": {
        "ecmaVersion": 7,
        "sourceType": "module",
        "ecmaFeatures": {
            "jsx": true
        }
    },
    "settings": {
        "react": {
            "pragma": "React"
        }
    }
}

```

**What did you do? Please include the actual source code causing the issue.**
```js
import React from 'react';
import PropTypes from 'prop-types';

const ItemList = (props) => {(
    <div>
      {
        props.articles.map(article =>
          <div
            key={article.id}>
            {article.name} {article.quantity}
          </div>
      } /!\
    </div>
  );
};

ItemList.propTypes = {
  articles: PropTypes.Array.isRequired
};

export default ItemList;
```

**What did you expect to happen?**
I got the error `Parsing error: Unexpected token, expected , (Fatal)` where I put the `/!\` symbol in my code

**Comment by gfelot** [comment]
My bad... i'm tired and forget an )
